.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This is a personal file, please remove it from the PR.

Reply:
okay